### ISSUE TITLE
Monolog 2.0 Support

### ISSUE THREAD

**Issue by browner12** [bugsnag/bugsnag-laravel#365]
Any chance we can get this package updated to work with Monolog v2?

I'm willing to give it a shot, but if someone is more familiar with Monolog and the v2 changes, might go a lot quicker.

**Comment by GrahamCampbell** [comment]
It should work already I think?

**Comment by GrahamCampbell** [comment]
Just the dependency needs updating in the composer.json by the looks of things.

**Comment by mikebronner** [comment]
Thanks @GrahamCampbell for making that PR. For those wanting to install bugsnag until the PR is merged:
```
composer require "bugsnag/bugsnag-laravel:dev-monolog-2@dev
```

**Comment by tomlongridge** [comment]
Fixed in 2.17.1
